src/django_components/commands/create.py

```diff
@@ -0,0 +1,220 @@
+import os
+import sys
+from textwrap import dedent
+from typing import Any
+
+from django.conf import settings
+from django.core.management.base import CommandError
+
+from django_components.util.command import CommandArg, ComponentCommand, style_success, style_warning
+
+
+class CreateCommand(ComponentCommand):
```

Comment:
This is what was previously `python manage.py startcomponent`. The `startcomponent` command is still available to the users, but the source of truth for the command logic is now defined indepedently of Django.